src/sentry/static/sentry/app/views/projectReleases/index.jsx

```diff
@@ -165,20 +170,36 @@ const ProjectReleases = createReactClass({
 
   renderEmpty() {
     const {environment} = this.state;
+    const {project} = this.context;
+    let anyProjectReleases = project.latestRelease;
     const message = environment
       ? tct("There don't seem to be any releases in your [env] environment yet", {
           env: environment.displayName,
         })
       : t("There don't seem to be any releases yet.");
-    return (
-      <EmptyStateWarning>
-        <p>{message}</p>
-        <p>
-          <a href={this.getReleaseTrackingUrl()}>
-            {t('Learn how to integrate Release Tracking')}
-          </a>
-        </p>
-      </EmptyStateWarning>
+
+    return anyProjectReleases === null ? (
+      <Hook
+        name="component:releases-tab-empty"
+        organization={this.context.organization}
+      />
+    ) : (
+      <div>
+        {anyProjectReleases !== null && (
+          <Hook
+            name="component:releases-tab-progress"
+            organization={this.context.organization}
+          />
+        )}
+        <EmptyStateWarning>
```

Comment:
do we need this in addition to the banner?

Reply:
I mean, if we show the banner we'll end up showing the EmptyStateWarning as well. If you want to avoid that, maybe use the HookOrDefault component we have?